### DISCUSSION TITLE
Improve accessibility of truncated pollen concentration description

### DISCUSSION THREAD

**Discussion by se34k** [breezy-weather/breezy-weather#996] [Ideas]
### Steps to reproduce

1. Set device language to German (or other language with rather long strings for "very low")
2. Open pollen card

### Expected behavior

Entire string "sehr niedrig" ("very low") or at least a long enough part to allow comprehension of the meaning of the description (e.g. "Sehr nied...") should be visible.

This could be implemented using either a marquee-style scrolling text or adding a menu that displays the entire description for the selected pollen type when tapped.

### Actual behavior

Only "Sehr ..." ("very ...") gets displayed, so users cannot see whether it means "very low" or "very high". 

While the dot on the left is green when it's low and red when it's high, people with impaired vision or color blindness aren't able to distinguish the colors and rely on the textual description to convey meaning - this poses an accessibility issue.
Additionally, new, unexperienced users who only see dots of one color might not realize the meaning of the color coding.

### Weather source used

Open-Meteo

### Breezy Weather version

5.2.1

### Android version

Android 11

### Device

Redmi Note 9S

### Other details

![IMG-20240421-WA0010~2](https://github.com/breezy-weather/breezy-weather/assets/76045565/7a7d40ed-fd1f-465e-96da-bd1157b0eb1d)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue.
- [X] I checked that my issue is not covered in the [Frequently Asked Questions](https://github.com/breezy-weather/breezy-weather/blob/main/HELP.md) document.
- [X] I have updated the app to **[version 5.2.1 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)** and I can reproduce the issue in this version.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance), unless my issue only happens with a specific language. I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
Isn't the content description working?
It should be described both on icon, and on the text itself of course.
